src/components/SearchJob.jsx

```diff
@@ -2,47 +2,70 @@ import React, { useState } from 'react';
 import {
 	Form,
 } from 'react-bootstrap';
-import cn from 'classnames';
+import styled from 'styled-components';
 
 function SearchJob() {
-	const [open] = useState(false);
+	const StyledForm = styled(Form)`
+		margin-top: 15px;
+		/* border: solid 1px rgb(39, 39, 39); */
+		border-radius: 5px;
+		width: 100%;
+		justify-content: center;
+`;
+
+	const StyledFormControl = styled(Form.Control)`
```

Comment:
This should likely be in its own component file as you'll want to use it elsewhere right?